packages/ui/src/views/layouts/main-content-layout.tsx

```diff
@@ -25,40 +24,35 @@ type MainContentLayoutProps = PropsWithChildren<{
   className?: string
 }>
 
-export function MainContentLayout({ children, useSidebar, withBreadcrumbs, className }: MainContentLayoutProps) {
-  const { isInset } = useTheme()
+export function MainContentLayout({ children, useSidebar, className, withBreadcrumbs }: MainContentLayoutProps) {
   const sidebarData = useSidebar?.()
   const isMobile = sidebarData?.isMobile
 
   return (
     <div
       className={cn(
-        'min-h-screen bg-cn-background-1',
+        'flex flex-col min-h-[calc(100vh-var(--cn-inset-layout-indent)*2)] my-[var(--cn-inset-layout-indent)] mr-[var(--cn-inset-layout-indent)] border rounded-md min-w-fit bg-cn-background-1',
         {
           'ml-[var(--cn-inset-layout-indent)]': isMobile,
           'min-h-[calc(100vh-var(--cn-breadcrumbs-height))]': withBreadcrumbs,
```

Comment:
Looks like we can kill off this line now since it's overridden in the next set of classes with the same condition.
```suggestion
```

Reply:
Make sense